packages/wallet-sdk/src/index.ts

```diff
@@ -15,7 +15,7 @@ declare global {
     /**
      * For CoinbaseWalletSDK, window.ethereum is `CoinbaseWalletProvider`
      */
-    ethereum?: unknown;
+    ethereum?: any;
```

Comment:
why are we using any here? Seems like we should be using the provider `CoinbaseWalletProvider`

Reply:
Thank you